coding-concepts.js

```diff
@@ -21,24 +21,24 @@ const greeter = (name) => {
 }
 // console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I dont think it will output anything because name is never defined
+// b) Verify and explain: "undefined" this is because "name" was never defined
```

Comment:
`name` is just a parameter, so when we write it in the parentheses, we are actually defining it right there. The argument is `"LEARN Student"`, which then takes the place of the parameter. We get `undefined` because the function does not have the `return` keyword.